package/debian/control

```diff
@@ -0,0 +1,118 @@
+Source: silx
+Maintainer: Debian Science Maintainers <debian-science-maintainers@lists.alioth.debian.org>
+Uploaders: Jerome Kieffer <jerome.kieffer@esrf.fr>,
+           Picca Frédéric-Emmanuel <picca@debian.org>
+Section: science
+Priority: extra
+Build-Depends: cython,
+               cython-dbg,
+               cython3,
+               cython3-dbg,
+               debhelper,
+               dh-python,
+               python-all-dev,
+               python-all-dbg,
+               python-h5py,
+               python-matplotlib,
+               python-matplotlib-dbg,
+               python-numpy,
+               python-numpy-dbg,
+               python-pyqt5,
+               python-pyqt5-dbg,
+               python-sphinx,
+               python-sphinxcontrib.programoutput,
+               python3-all-dev,
+               python3-all-dbg,
+               python3-matplotlib,
+               python3-matplotlib-dbg,
+               python3-numpy,
+               python3-numpy-dbg,
+               python3-pyqt5,
+               python3-pyqt5-dbg,
+               python3-sphinx,
+               python3-sphinxcontrib.programoutput,
+Standards-Version: 3.9.6
+Vcs-Browser: https://anonscm.debian.org/cgit/debian-science/packages/silx.git
+Vcs-Git: git://anonscm.debian.org/debian-science/packages/silx.git
+Homepage: http://forge.epn-campus.eu/projects/azimuthal
+X-Python-Version: >= 2.7
+X-Python3-Version: >= 3.4
+
+#Package: silx
+#Architecture: all
+#Depends: ${misc:Depends},
+#         ${python:Depends},
+#         ${shlibs:Depends},
+#         python3-silx (>= ${source:Version})
+#Description: Toolbox for X-Ray data analysis - Executables
+
+Package: python-silx
+Architecture: any
+Section: python
+Depends: ${misc:Depends},
+         ${python:Depends},
+         ${shlibs:Depends},
+         python-h5py,
+         python-pyqt5
```

Comment:
Add python-matplotlib and python3-matplotlib to Depends as plot will depend on it.